disk/check.py

```diff
@@ -61,6 +62,7 @@ def _load_conf(self, instance):
             instance.get('tag_by_filesystem', False))
         self._all_partitions = _is_affirmative(
             instance.get('all_partitions', False))
+        self._device_tag_re = instance.get('device_tag_re', [])
```

Comment:
default type in the example config is a dict.

Reply:
The example config shows a list of dicts. Is there a better way to define this in YAML? It is a bit weird because I'm using regexes as the dictionary keys.

Reply:
we could use a yaml dict instead of a list:

```yaml
device_tag_re:
  vagrant.*: device_type:virtual
  /dev/.*: device_type:dev,tag:two
```

which will translate to:

```python
{
  'device_tag_re':
    {
      '/dev/.*': 'device_type:dev,tag:two',
      'vagrant.*': 'device_type:virtual',
    }
}
```